units/host_stats.go

```diff
@@ -51,18 +51,31 @@ func NewHostStatsJob(ts string) amboy.Job {
 func (j *hostStatsJob) Run(_ context.Context) {
 	defer j.MarkComplete()
 
-	counts, err := host.CountInactiveHostsByProvider()
+	if j.logger == nil {
+		j.logger = logging.MakeGrip(grip.GetSender())
+	}
+
+	inactiveHosts, err := host.CountInactiveHostsByProvider()
 	if err != nil {
 		j.AddError(errors.Wrap(err, "error aggregating inactive hosts"))
 		return
 	}
+	j.logger.Info(message.Fields{
+		"message": "count of decommissioned/quarantined hosts",
+		"counts":  inactiveHosts,
+	})
 
-	if j.logger == nil {
-		j.logger = logging.MakeGrip(grip.GetSender())
+	taskSpawned, err := host.FindAllHostsSpawnedByTasks()
+	if err != nil {
+		j.AddError(errors.Wrap(err, "error finding hosts spawned by tasks"))
+		return
+	}
+	hostIds := []string{}
+	for _, h := range taskSpawned {
+		hostIds = append(hostIds, h.Id)
 	}
-
 	j.logger.Info(message.Fields{
-		"message": "count of decommissioned/quarantined hosts",
-		"counts":  counts,
+		"message": "hosts spawned by tasks",
+		"hosts":   hostIds,
```

Comment:
I feel like we also want to log what task spawned them (which will be set in the `User` field) and their task and build scopes, so the information is available immediately in splunk without having to look up the hosts in the UI or DB.

Reply:
Added more fields